WebService/ComparisonEngineUI/ComparisonEngineUI/Data/RestService.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using WebService.Base_Classes;
+
+namespace ComparisonEngineUI.Data
+{
+    class RestService
+    {
+        HttpClient client;
+
+        public RestService()
+        {
+            client = new HttpClient();
```

Comment:
does client need any explicit disposing or does it get picked up by garbage collector?

Reply:
this shouldn't be a problem because we can use "using" when we want to get/save data.